crazyflie_examples/crazyflie_examples/set_param.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+from crazyflie_py import *
+
+
+def main():
+    Z = 1.0
+    
+    swarm = Crazyswarm()
+    timeHelper = swarm.timeHelper
+    allcfs = swarm.allcfs
+
+    # disable LED (one by one)
+    for cf in allcfs.crazyflies:
+        cf.setParam("led.bitmask", 128)
```

Comment:
This doesn't disable the LEDS, this is actually the default value already. So 128 should be 0